### PR TITLE
Admin item management #10

### LINKED ISSUE #10: User Story 59, Admin can manage items on behalf of a merchant
```
As an admin user
When I visit a merchant's profile page
I can click on the merchant's items link
And have access to all functionality the merchant does, including
- adding new items
- editing existing items
- enabling/disabling/deleting items

All content rules still apply (eg, item name cannot be blank, etc)
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #19

child of #79

depends on #18

depends on #14

depends on #12

### PR DESCRIPTION
**Pull request by allieIsOkay**
- Adds tests for admin ability to add items
- Adds tests for admin ability to edit items
- Adds tests for admin ability to delete unordered items
- Adds tests for admin ability to enable disabled items
- Adds tests for admin ability to disable ordered items
- Adds admin item functionality
- Updates admin namespace item routing
- closes #10

### CODE REVIEW

File: app/controllers/admin/items_controller.rb [hunk 1]
```diff
@@ -1,6 +1,66 @@
 class Admin::ItemsController < Admin::BaseController
   def index
-    @items = Item.where(user_id: params[:merchant_id])
+    @merchant = User.find(params[:merchant_id])
```
Comment (timnallen) [inline app/controllers/admin/items_controller.rb:3]: Good catch here.

**Review by timnallen — APPROVED** [review]
Great work here!

**Review by teresa-m-knowles — APPROVED** [review]
Looks really good, Peter

**Review by csvlewis — APPROVED** [review]
Good work, Peter! I like the one-line conditional statements that you put in the item index view.
